public/gameList.php

```diff
@@ -23,6 +23,10 @@
     $offset = 0;
 }
 
+if ($consoleIDInput > 0) {
+    return redirect()->route('system.game.index', ['system' => $consoleIDInput]);
+}
+
```

Comment:
I notice "All Games" hasn't been converted. I'm guessing that's intentional?

Reply:
Intentional, as we currently don't support pagination and I figured such an implementation could be hairy enough to justify its own PR.